publiq-ui/pub-wrapper.stories.mdx

```diff
@@ -0,0 +1,26 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubWrapper, { ButtonVariant } from './pub-wrapper.vue';
+
+<Meta title="pub-wrapper" component={PubWrapper} />
+
+export const Template = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    components: { PubWrapper },
+    template: '<pub-wrapper v-bind="$props">Wrapped content</pub-wrapper>',
+  };
+};
+
+# Variants
```

Comment:
I think you can leave out the `# Variants` here for components that don't have a `variant` prop